puzzlegenerator/puzzlegenerator_ui/src/main/java/com/flurnamenpuzzle/generator/ui/view/PuzzleGeneratorView.java

```diff
@@ -115,6 +116,8 @@ public void update(Observable observable) {
 		PuzzleGeneratorModel model = (PuzzleGeneratorModel) observable;
 		Steps currentStep = model.getCurrentStep();
 		String currentStepId = currentStep.getId();
+		String imagePath = currentStep.getImagePath();
+		this.steps.setIcon(new ImageIcon(this.getClass().getResource(imagePath)));

```

Comment:
Die Bilder sollten eigentlich in einem eigenen Source Ordner  (optimalerweise mit dem Namen resources) liegen. Du kannst im service Modul nachschauen, wie das für die Tests gelöst ist, dort greifen wir auf Files im Ordner src/test/resources zu. Genau gleich sollten die Bilder unter src/main/resources liegen.